app/models/event.rb

```diff
@@ -178,7 +178,8 @@ def require_location
   # Ensures that the event added is unique
   # Checks for a duplicate event with the properties dtstart, summary, and calendar_id
   def unique_event
-    return unless Event.where(dtstart: dtstart,
+    return unless Event.where(uid: uid,
```

Comment:
if the `uid` is unique do we still need to check `dstart`,`summary`, `calendar_id` ?

Reply:
i felt making the scope tighter maintains backward compatibility with the rest of the event source importers whilst adding extra checks. 
in an ideal world UID would be unique per event (per calendar) and we could rely on it to determine uniqueness but determining if that is the case would be some effort.

Reply:
thanks for explaining, good to know that might not be the case.